cmd/containerd/main.go

```diff
@@ -120,11 +120,13 @@ high performance container runtime
 		)
 		switch runtime {
 		case "runc":
+			logrus.Info("containerd: run with runc executor")
```

Comment:
Updated, I'll take a look at logrus formatters tomorrow, see if we can do something with that, thanks.